redash/query_runner/pg.py

```diff
@@ -182,6 +184,8 @@ def run_query(self, query, user):
 class Redshift(PostgreSQL):
     default_doc_url = ("http://docs.aws.amazon.com/redshift/latest/"
                        "dg/cm_chap_SQLCommandRef.html")
+    data_source_version_query = "select version()"
+    data_source_version_post_process = "split by space take last"
```

Comment:
These seem easily typo'd. Feel free to merge as-is but here's a suggestion that could use Python syntax checker to catch errors:

```
POST_PROCESS_NONE = 1
POST_PROCESS_SPLIT_SECOND = 2
POST_PROCESS_SPLIT_LAST = 3
```
Then use those constants:
```
data_source_version_post_process = constants.POST_PROCESS_SPLIT_SECOND
```
And further up used in the conditional:
```
if self.data_source_version_post_process == constants.POST_PROCESS_SPLIT_SECOND:
    # ...
```